__tests__/app.test.js

```diff
@@ -39,3 +40,61 @@ describe("/api/categories", () => {
     });
   });
 });
+describe("/api/reviews", () => {
+  describe("GET /api/reviews/:review_id", () => {
+    it("status: 200, response with an array of review objects with its properties  ", () => {
+      const review_id = 7;
+      
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then((res) => {
+          expect(res.body.review).toBeInstanceOf(Object);
+          // expect(res.body.review).toHaveLength(1);
+          expect(res.body.review).toEqual(
+            expect.objectContaining({
+              review_id: 7,
+              title: "Mollit elit qui incididunt veniam occaecat cupidatat",
+              designer: "Avery Wunzboogerz",
+              owner: "mallionaire",
+              review_img_url:
+                "https://images.pexels.com/photos/278888/pexels-photo-278888.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260",
+              review_body:
+                "Consectetur incididunt aliquip sunt officia. Magna ex nulla consectetur laboris incididunt ea non qui. Enim id eiusmod irure dolor ipsum in tempor consequat amet ullamco. Occaecat fugiat sint fugiat mollit consequat pariatur consequat non exercitation dolore. Labore occaecat in magna commodo anim enim eiusmod eu pariatur ad duis magna. Voluptate ad et dolore ullamco anim sunt do. Qui exercitation tempor in in minim ullamco fugiat ipsum. Duis irure voluptate cupidatat do id mollit veniam culpa. Velit deserunt exercitation amet laborum nostrud dolore in occaecat minim amet nostrud sunt in. Veniam ut aliqua incididunt commodo sint in anim duis id commodo voluptate sit quis.",
+              category: "social deduction",
+              created_at: "2021-01-25T11:16:54.963Z",
+              votes: 9,
+            })
+          );
+        });
+    });
+  describe("GET /api/reviews/:review_id ERRORS", () => {
+    it("status:400, should respond with error message bad request when given invalid data type", () => {
+      return request(app)
+      .get("/api/reviews/:review_id")
```

Comment:
74 - careful with the address here.  You are correct, this is an invalid data type (string), but it is slightly confusing because it looks like it may have been a mistake writing :review_id since it looks like a parametric endpoint identifier.  To be safe it could be good to use something that is 'obviously' incorrect, for example /api/reviews/NOTAREVIEWID